python/py-corner/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-corner
+version             2.0.1
+
+categories-append   science
+maintainers         {danfm.ca:dan @dfm} {ligo.org:duncan.macleod @duncanmmacleod} openmaintainer
+
+platforms           darwin
+license             GPL-3
+
+description         Make some beautiful corner plots of samples
+long_description    Make some beautiful corner plots of samples
```

Comment:
I'm going to merge this, but generally, please make long_description more descriptive than this. It's intended to try to explain the function of the package enough that someone will know whether they're interested in it.